### ISSUE TITLE
FP for CA2208

### ISSUE THREAD

**Issue by Evangelink** [dotnet/roslyn-analyzers#6925]
### Analyzer

**Diagnostic ID**: [CA2208](https://learn.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca2208)

### Analyzer source

**Version**: SDK 8.0.100-preview.7.23376.3

### Describe the bug

CA2208 is reported but the construction of the exception is valid and correct.

### Steps To Reproduce

```
throw new ArgumentException($"{nameof(value)}[2] must be a ':'", nameof(value));
```

## Additional context

[Original source code](https://github.com/dotnet/iot/blob/main/src/devices/Display/Large4Digit7SegmentDisplay.cs#L256)

**Comment by mpidash** [comment]
This seems to be the same regression as #6863, described here: https://github.com/dotnet/roslyn-analyzers/issues/6863#issuecomment-1684930305

Reverting to exact matches when checking the message argument should fix both issues (and keeping the more lenient match check for the parameter name argument).

**Comment by Evangelink** [comment]
Oh thanks for catching the duplicate. Closing this one.
